pkg/cli/alpha/config-gen/cmd.go

```diff
@@ -254,7 +254,7 @@ kubebuilder alpha config-gen kubebuilderconfiggen.yaml patch1.yaml patch2.yaml
 #
 
 # install the latest kustomize
-GO111MODULE=on go get sigs.k8s.io/kustomize/kustomize/v4
+curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
```

Comment:
If we only do that we will install the latest version always. We need to pass the version to the script.

Reply:
The documentation line I'm replacing did not have a version, it just installed the latest v4 release. Should we add a specific version here?

Reply:
Yes. We need to clear to the users that they ought to use the compatible/supported version. 
E.g: https://github.com/kubernetes-sigs/kubebuilder/pull/2486/files#diff-2ce1a29a9c5ec9b4069ab522cad03d59a54521b8311fcaab953a551be8aa5b23R116

See: https://book.kubebuilder.io/quick-start.html#versions-and-supportability
<img width="884" alt="Screenshot 2022-02-04 at 17 39 28" src="https://user-images.githubusercontent.com/7708031/152599997-b47e2744-5fd1-459f-8ba2-64331062d6bd.png">

In this case e.g. if users use v4 for kustomize then, it schema will not work with k8s versions < 1.22. See: https://github.com/kubernetes-sigs/kubebuilder/pull/2498#issuecomment-1026772984